torch/_decomp/decompositions.py

```diff
@@ -2010,37 +2015,53 @@ def upsample_bilinear2d(
     out_w = sym_float(output_size[1])
 
     # Calculate horizontal and vertical scaling factor
-    # TODO: Figure out if scales_h/scales_w matters here
+    # See compute_scales_value and area_pixel_compute_scale in core.
     if out_h > 1:
         if align_corners:
```

Comment:
Note: checked in different order in `area_pixel_compute_scale`: first align, then out.

The logic in the decomp kinda makes sense: why div by `out` if `out <= 1`? Not sure what's right here, so didn't touch this.